gtest-parallel

```diff
@@ -424,4 +470,6 @@ if options.print_test_times:
               if times.get_test_time(test_binary, test) is not None)
   for (time_ms, test_binary, test) in ts:
     print "%8s %s" % ("%dms" % time_ms, test)
+if options.dump_json_test_results:
+  test_results.dump_to_file(options.dump_json_test_results)
```

Comment:
Or call it .close() on CollectTestResults, and it writes on close, that makes sense to me. @kwiberg probably has a better method name, but I think CollectTestResults should hold onto the file handle.